cmd/util.go

```diff
@@ -384,3 +384,36 @@ func shouldAuditCSC(podSpec PodSpecV1, container ContainerV1) bool {
 	}
 	return false
 }
+
```

Comment:
I don't think we need two different functions in `getContainerOverrideLabelReason` and `isDefinedContainerOverrideLabel` and same for Pods. Why don't we just return true or false from `getContainerOverrideLabelReason` currently defined and rename it to something like `containerOverrideLabelValid` ?

Reply:
nicee this is a good idea

Reply:
actually we need the overridelabelreason for the result/occurrence. i think its a bit neater to have two functions since i am using both in the code

Reply:
I don't see how it's useful for result/occurrence.. can you point out where? I also disagree with the neater part, `isDefinedContainerOverrideLabel` is literally `getContainerOverrideLabelReason` function where you return bool... if you are not using OverrideLabelReason anywhere outside checking if it exists and it matches I don't see the point of fetching the reason instead

Reply:
`else if container.SecurityContext == nil || container.SecurityContext.AllowPrivilegeEscalation == nil || *container.SecurityContext.AllowPrivilegeEscalation == true {
		occ := Occurrence{
			container: container.Name,
			id:        ErrorAllowPrivilegeEscalationTrueAllowed,
			kind:      Warn,
			message:   "Allowed AllowPrivilegeEscalation to be set as true",
			metadata:  Metadata{"Reason": prettifyReason(reason)},
		}`

Reply:
it is used in check-xxxx functions. I dont really see why are u against it, but if you really insist, i could just check reason != ""

Reply:
Ah it's used in Metadata, I guess we can leave it there then, I have a different suggestion in that case. Why don't we have one function and return string and bool from it? It would be much more elegant. Return reason as a string and bool dependent on whether label's defined or not?

Reply:
Done